lms/djangoapps/onboarding_survey/forms.py

```diff
@@ -205,3 +209,103 @@ def clean(self):
                     'founding_year',
                     "You entered an invalid year format. Please enter a valid year with 4 digits."
                 )
+
+
+class RegModelForm(forms.ModelForm):
+    """
+    Model form for extra fields in registration model
+    """
+    organization_name = forms.CharField(
+        max_length=255,
+        label='Organization Name',
+        widget=forms.TextInput(
+            attrs={'placeholder': 'Organization Name'}
+        ),
+        initial='Organization Name'
+    )
+
+    confirm_password = forms.CharField(
+        label='Confirm Password',
+        widget=forms.PasswordInput(attrs={'placeholder': 'Confirm Password'}),
+        initial='Confirm Password'
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(RegModelForm, self).__init__(*args, **kwargs)
+        self.fields['first_name'].initial = 'First Name'
+        self.fields['last_name'].initial = 'Last Name'
+        self.fields['org_admin_email'].initial = 'Organization Admin Email'
+
+        self.fields['first_name'].error_messages = {
+            'required': 'Please enter your First Name.',
+        }
+
+        self.fields['last_name'].error_messages = {
+            'required': 'Please enter your Last Name.',
+        }
+
+        self.fields['organization_name'].error_messages = {
+            'required': 'Please select your Organization.',
+        }
+
+        self.fields['confirm_password'].error_messages = {
+            'required': 'Please enter your Confirm Password.',
+        }
+
+    class Meta:
+        model = ExtendedProfile
+
+        fields = (
+            'confirm_password', 'first_name', 'last_name',
+            'organization_name', 'is_currently_employed', 'is_poc', 'org_admin_email',
+        )
+
+        labels = {
+            'is_currently_employed': "Check here if you're currently not employed.",
+            'is_poc': 'Are you the Admin of your organization?',
+            'org_admin_email': 'If you know who should be the Admin, please add their email address below. We\'ll send'
+                               ' them an email inviting them to join the platform as the organization admin.',
+        }
+
+        widgets = {
+            'first_name': forms.TextInput(attrs={'placeholder': 'First Name'}),
+            'last_name': forms.TextInput(attrs={'placeholder': 'Last Name'}),
+            'is_poc': forms.RadioSelect(choices=((1, 'Yes'), (0, 'No'))),
+            'org_admin_email': forms.EmailInput(attrs=({'placeholder': 'Email'}))
+        }
+
+        serialization_options = {
+            'confirm_password': {'field_type': 'password'},
+            'org_admin_email': {'field_type': 'email'}
+        }
+
+        initial = {
+            'first_name': 'First Name',
+            'last_name': 'Last Name'
+        }
+
+    def save(self, commit=True):
+        extended_prof_model = super(RegModelForm, self).save(commit=False)
```

Comment:
Could you please rename it to `extended_profile`?

Reply:
Done